src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS
+            = {"red", "green", "blue", "yellow", "pink", "black", "white"};
+
+    public String getRandomColor() {
+        Random random = new Random();
```

Comment:
The Random object should be a class-level variable instead of being instantiated every time the getRandomColor() method is called. This is because creating a new Random instance every time may not be efficient and the use of a single Random instance is thread-safe for this use case.